### PR TITLE
feat: Localization feature was added #45

### PR DESCRIPTION
**Pull request by anton-marchenko**
### Description

[Localization](https://github.com/nhnent/tui.image-editor/issues/45) [feature](https://github.com/nhnent/tui.image-editor/issues/85) was added. Please, let me know if I need to change something into my PR.
Maybe tests should be added.
Also, maybe this feature must to change current npm version of tui.image-editor.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by seonim-ryu** [comment]
Can one of the admins verify this patch?

**Comment by anton-marchenko** [comment]
@junghwan-park
@jungeun-cho
Anyone, give me feedback, please!

File: src/js/ui/locale/locale.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        this._locale = locale;
+    }
+
+    /**
+     * localize message
+     * @param {string} message - message who will be localized
+     * @returns {string}
+     */
+    localize(message) {
+        return (this._locale[message]) ? this._locale[message] : message;
```
Comment (dongsik-yoo) [inline src/js/ui/locale/locale.js]: Is this parenthesis necessary? I think it's better using  short-circuit evaluation than conditional ternary operator.
```js
return this._locale[message] || message;
```

File: README.md [hunk 2]
```diff
@@ -247,12 +247,18 @@ Add dependencies & initialize ImageEditor class with given element to make an im
 ```javascript
 var ImageEditor = require('tui-image-editor');
 var blackTheme = require('./js/theme/black-theme.js');
+var locale_ru = { // override default English locale to your custom
```
Comment (dongsik-yoo) [inline README.md]: I think we need a document of a full key list. How about [/docs/Basic-Tutorial.md](https://github.com/nhnent/tui.image-editor/blob/master/docs/Basic-Tutorial.md)? 
@jinwoo-kim-nhn Anh idea?

File: src/js/ui/locale/locale.js (hunk 1 above)
Comment (anton-marchenko) [reply src/js/ui/locale/locale.js]: @dongsik-yoo, It is a good idea. I refactored it.

**Comment by dongsik-yoo** [comment]
The documentation issue is on #110 .
Thank you @anton-marchenko . Really great helpful!.

File: README.md (hunk 2 above)
Comment (anton-marchenko) [reply README.md]: @dongsik-yoo, I will make doc today.

**Pull request merged**